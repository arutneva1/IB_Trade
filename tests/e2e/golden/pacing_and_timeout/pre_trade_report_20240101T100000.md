NetLiq: 17500.00
Cash USD: 10000.00
Cash Buffer: 100.00

| symbol | target_pct | current_pct | drift_bps | price | dollar_delta | share_delta | side | est_notional | reason |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AAA | 66.67 | 28.74 | 3793.10 | 100.00 | 6637.93 | 66.3793 | BUY | 6637.93 |  |
| BBB | 33.33 | 14.37 | 1896.55 | 50.00 | 3318.96 | 66.3792 | BUY | 3318.96 |  |
| TOTAL | 100.00 | 43.11 | 5689.65 |  | 9956.89 |  |  | 9956.89 |  |
