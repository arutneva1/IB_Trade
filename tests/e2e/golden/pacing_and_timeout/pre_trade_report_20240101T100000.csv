NetLiq,17500.00
Cash USD,10000.00
Cash Buffer,100.00

symbol,target_pct,current_pct,drift_bps,price,dollar_delta,share_delta,side,est_notional,reason
AAA,66.67,28.74,3793.1,100.0,6637.93,66.3793,BUY,6637.93,
BBB,33.33,14.37,1896.55,50.0,3318.97,66.3794,BUY,3318.97,
TOTAL,100.0,43.11,5689.65,,9956.9,,,9956.9,
